memory/ndctl.py

```diff
@@ -405,6 +405,30 @@ def test_namespace_modes(self):
         if failed_modes:
             self.fail("Namespace for %s mode failed!" % failed_modes)
 
+    def test_namespace_devmap(self):
+        """
+        Test metadata device mapping option with a namespace
+        """
+        region = self.get_default_region()
+        m_map = self.params.get('map', default='mem')
+        size_align = self.get_size_alignval()
+        namespace_size = self.params.get('size', default=134217728)
```

Comment:
@harish-24 just a suggestion: 
can 134217728 be expanded for better understanding, or can it be documented ?